packs/ocm-klusterlet-v0.13.2/charts/ocm-klusterlet/templates/service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: klusterlet
+  namespace: open-cluster-management
```

Comment:
The pack allows user to change the namespace through pack.namespace. But, the namespaces are hard coded in some of the templates. As a standard, the helm chart templates should use the release namespace to allow user customizations.